unity/EditorPlugin/AssetPostprocessors/CsprojAssetPostprocessor.cs

```diff
@@ -264,7 +264,7 @@ private static bool ApplyManualCompilerSettings([CanBeNull] string configFilePat
 
       var changed = false;
       
-      // Unity always sets AllowUnsafeBlocks in 2017.1+
+      // Unity sets AllowUnsafeBlocks in 2017.1+ depending on Player settings or asmdef
       // Strictly necessary to compile unsafe code
       // https://github.com/Unity-Technologies/UnityCsReference/blob/2017.1/Editor/Mono/VisualStudioIntegration/SolutionSynchronizationSettings.cs#L119
```

Comment:
Is this comment change correct? The link shows a hard-coded `<AllowUnsafeBlocks>true</AllowUnsafeBlocks>`, it doesn't appear dependent on anything

Reply:
@citizenmatt 
![image](https://user-images.githubusercontent.com/1482681/47842405-d318ad00-ddbc-11e8-856e-592d33f87166.png)